cupy_setup_build.py

```diff
@@ -214,6 +217,7 @@
 
     MODULES.append({
         'name': 'cutensor',
+        'required': True,
```

Comment:
cuTENSOR is an optional. (sorry I failed to list it up in my original issue :bow:_

Reply:
Fixed!